src/server/pkg/worker/master.go

```diff
@@ -432,11 +432,13 @@ func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *grpc
 						return fmt.Errorf("error from connection pool: %v", err)
 					}
 					workerClient := NewWorkerClient(conn)
+					protolion.Infof("issuing Process() request for job (%v)\n", jobInfo)
```

Comment:
I know this one seems a bit redundant w the logs that are within the `Process()` command ... but I'm seeing this issue w a customer where the worker `Process()` call completes ... but we don't get the response. These logs will help me identify when this happens (it's intermittent).